### ISSUE TITLE
10 concurrency limit not working, TypeError

### ISSUE THREAD

**Issue by tobiasmuehl** [supercharge/promise-pool#29]
```
const PromisePool = require('@supercharge/promise-pool')
await PromisePool
		.for([1, 2, 3, 4])
		.process(async c => console.log(c))
```

```
TypeError: "concurrency" must be a number, 1 or up. Received "undefined" (undefined)
    at PromisePoolExecutor.validateInputs (/home/t/project/node_modules/@supercharge/promise-pool/dist/promise-pool-executor.js:103:19)
    at PromisePoolExecutor.start (/home/t/project/node_modules/@supercharge/promise-pool/dist/promise-pool-executor.js:89:36)
    at PromisePool.process (/home/t/project/node_modules/@supercharge/promise-pool/dist/promise-pool.js:89:14)
    at /home/t/project/script.ts:42:4
    at step (/home/t/project/script.ts:36:23)
    at Object.next (/home/t/project/script.ts:17:53)
    at fulfilled (/home/t/project/script.ts:8:58)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
```

**Comment by marcuspoehls** [comment]
@tobiasmuehl Good catch! I didn’t notice this while working with the library. Thank you for reporting this!

Do you want to submit a pull request fixing the issue?

**Comment by marcuspoehls** [comment]
I’ve published version `1.6.2` on NPM. Should fix this issue.

@tobiasmuehl Tobias, please update to `1.6.2`.
